v0/destinations/mautic/transform.js

```diff
@@ -204,7 +277,17 @@ const process = async event => {
       );
       break;
     default:
-      throw new CustomError(`Message type ${messageType} not supported.`, 400);
+      throw new TransformationError(
```

Comment:
a suggestion, if you have multiple errors, you can create one new `errors.ts` file and define all the errors there, import the file and return that error instance from the designated places. It will make your code more readable :)

Reply:
We have to implement this for all the existing destinations. We can consider this in a separate task.

Reply:
@Gauravudia Lets track this somewhere, if this can be prioritised.

Reply:
sure @yashasvibajpai

Reply:
@crownedlake still DESTINATION param changes. one more way, we can move few defaults to the error class implementation itself & use Constants for Error messages. Check out this page where we are discussing on these & add your comments - https://www.notion.so/rudderstacks/Transformer-Alerting-Objectives-dcc0dab0bd8640f5bb46d1da45bc69ec

@Gauravudia we can merge this PR & once we finalize the next set of changes, we can pick up. That's the reason to hold off refactoring error classes for next set of destinations.